LondonBritishLibrary/orient/BLorient2263.xml

```diff
@@ -0,0 +1,219 @@
+                                ልብ፡ ቃል፡ ወእስትንፋስ፡ እንዳይለያዩ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አይለያዩም። 
+                                እምቅድመ፡ <pb n="1rb"/> ዓለም፡ እንዴኽ፡ ነበሩ፡ ዓለም፡ ከኀለፈም፡ እንዴኽ፡ ይኖላናሉ፡ ይኽም፡ ይኽት፡ ፀሐይ፡ ም፡ ሐንድ፡ ነው፡ በሐንደነቱ፡ ግን፡ 
+                                ሶስትንት፡ አለበት፡ አብ፡ ክበቡን፡ ይመስል፡ ወልድ፡ ብርሃኑ፡ ይመስል፡ መንፈስ፡ ቅዱስ፡ ሙቀቱ፡ ይመስል። አብነም፡ ስለመሰሉት፡ በክበቡ፡ እንደ፡ 
+                                ፀሐ<pb n="1va"/>ይ፡ ውሱን፡ ክበብ፡ አይዶለውም፡ የሰማዩም፡ ሰማይ፡ የዕመቁ፡ ም፡ ዕመቅ፡ የጽንፉ፡ ም፡ ጽንፍ፡ አይወሰነውም፡ ወልድነም፡ በብርሃኑ፡ 
+                                ስለመሰሉት፡ ውሱን፡ ብርሃን፡ አይዶለውም፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus target="#2r"/>
+                            <title ref="LIT3013RepCh293" type="incomplete">Hymn to the Church</title>
+                            <note>The beginning is missing. The poem is apparently inserted in the middle of <ref target="#ms_i1"/>.</note>
+                            <explicit xml:lang="gez">ለኪ፡ ዘአፈይከኡ፡ ስብሐተ።
+                                አምሳለ፡ ቍርባን፡ ተወካፊ፡ ሊተ።
+                                ወተረፈ፡ ቃልየ፡ ወትረ፡ ረስዪ፡ መሥዋዕተ።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ በኆልቈ፡ እስትንፋስ፡ ወነገር።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ በኆልቈ፡ አዕፅምት፡ ወጸጕር።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ ዘአብሐኪ፡ እግዚአብሔር።
+                                በላዕለ፡ ሥጋሁ፡ ቅዱስ፡ ወደሙ፡ ክቡር።</explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="6ra" to="12vb"/>
+                            <title>Turkish and Arabic vocabulary</title>
+                            <note>As a whole indicated by: <foreign xml:id="ti"><title>ነገረ፡ ትርኵ። ዓ።</title></foreign> written on the upper
+                                margin.</note>
+                        <msItem xml:id="ms_i3.1">
+                            <locus target="#6ra"/>
+                            <title ref="NAR0184ForeignWords">Tigrinya-Turkish Vocabulary</title>
+                            <note>Containing only eight words.</note>
+                            <incipit xml:lang="ti">ነዓ፡ ብሂል፡ ዢል፤
+                                ተቀመጥ፡ ብ፡ ውቱር፤
+                                ሀባ፡ ብ፡ ዠትር፤
+                                አልቦን፡ ብ፡ ያክትር፤
+                                ሀሎ፡ ብ፡ ዮል፤
+                                ኪድ፡ ብ፡ ቢናዓደም፤
+                                መጽአ፡ ብ፡ ገልድ፤
+                                ኬደ፡ ብ፡ ደልዳል።</incipit>
+                            <textLang xml:lang="ti"/>
+                            </msItem>
+                            <msItem xml:id="ms_i3.2">
+                                <locus from="6ra" to="12vb"/>
+                                <title ref="NAR0184ForeignWords">Tigrinya-Arabic Vocabulary</title>
+                                <note>Containing several words on <locus target="6ra"/> with a special ancient form of the letter 
```

Comment:
Both this and the Turkish word list are edited in the article by Littmann that Strelcyn refers to. Maybe it would be worthwhile including that information?

https://menadoc.bibliothek.uni-halle.de/dmg/periodical/titleinfo/118035

Reply:
Yes, infact, I forgot to add the bibl-information. I will do in a minute!